source/state_representation/README.md

```diff
@@ -28,45 +28,33 @@ By default, quaternions are normalized on setting, therefore:
 s2.set_orientation(Eigen::Quaterniond(1, 1, 0, 0));
 ```
 
-The state variables are also grouped in `pose` (`position` and `orientation`), `twist` (`linear_velocity` and `angular_velocity`), `accelerations` (`linear_acceleration` and `angular_acceleration`) and `wrench` (`force` and `torque`).
+The state variables are also grouped in `pose` (`position` and `orientation`), `twist` (`linear_velocity` and
+`angular_velocity`), `accelerations` (`linear_acceleration` and `angular_acceleration`) and `wrench` (`force` and `torque`).
 Getter and setters are also implemented to do those bulk operations.
 
 ```cpp
-s2.set_twist(Eigen::VectorXd::Random(6))
+s2.set_twist(Eigen::VectorXd::Random(6));
 ```
 
 Note that for `pose`, it will be a `7d` vector (3 for `position` and 4 for `orientation`):
 
 ```cpp
-s2.set_pose(Eigen::VectorXd::Random(7))
+s2.set_pose(Eigen::VectorXd::Random(7));
 ```
 
 ### States operations
 
-Basic operations between frames such as addition, substractions, scaling are defined, and applied on all the state variables.
+Basic operations between frames such as addition, subtractions, scaling are defined, and applied on all the state variables.
 
 ```cpp
 StateRepresentation::CartesianState s1("a"); // reference frame is world by default
 StateRepresentation::CartesianState s2("b");
+double lamda = 0.5
 
-// for this operation to be valid both s1 and s2
+// for those operation to be valid both s1 and s2
 // should be expressed in the same reference frame
```

Comment:
I feel this line needs more emphasis, potentially worth repeating in the section header above. 

And, perhaps you can make it clear that these are transformations and therefore not commutative. For example (although perhaps the phrasing can still be improved):

For two states s1 and s2 in the same base reference frame, the addition `s1 + s2` is equivalent to the transformation from the base frame to s1 followed by the transformation from the base frame to s2. Subtraction of a state corresponds to the inverse transformation.

Reply:
very good point thank you